AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -189,6 +191,25 @@ - (void)setOpaque:(BOOL)opaque
 @synthesize delegate = _delegate;
 
 #pragma mark -
+@dynamic textView;
+
+- (UITextView *)textView
+{
+  ASDisplayNodeAssertMainThread();
+  if (!_textView) {
```

Comment:
This will then just be !_textKitComponents.textView,

and if it doesn't exist, just set _textKitComponents.textView = alloc / init.

Then just quickly check the various areas in the class that use _textKitComponents to ensure that in all cases, this accessor method will be called at least once before the textView is used, so that it is set up properly.